webapp/views/wpt-app.js

```diff
@@ -386,6 +390,21 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
     return true;
   }
 
+  showAddEdgeBackLink(queryParams) {
```

Comment:
Another case where we can't add Edge back is the links for PR checks, such as https://wpt.fyi/?sha=ec5f766aec&label=pr_head. There is no Edge run to add back in that case.

The ideal behavior would be to detect the cases where the runs we are showing are inferred from the default product set, and where adding Edge to that default would still result in some runs to show. They might not be for the same commit in the case of aligned runs, but an additional run from Edge should appear.

Implementing this would presumably requiring using `/api/runs` a few times, so are there simpler approximations of this that get it right in most cases?